builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -425,10 +425,6 @@ node() {
                   error "Deployment targets are not defined"
               }
 
-              if (domain != "jazz") {
```

Comment:
Why is this removed?

Reply:
Actually during resolving merge conflicts with develop, accidentally this line got duplicated and I missed it. So I removed this here since it's already present just few lines above the code.

Reply:
Yep, just checked. Resolving this.